DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/packages.lock.json

```diff
@@ -404,7 +434,7 @@
         "dependencies": {
           "Serilog.Extensions.Logging": "[7.0.0, )",
           "Speckle.Autofac": "[2.0.999-local, )",
-          "Speckle.Core": "[2.0.999-local, )"
+          "Speckle.Core": "[3.0.1-alpha.11, )"
```

Comment:
Should we have some rules around what we should pick up? i.e. accept minor versions and bug fixes? Maybe this isn't the right place and we perhaps don't need to do it right away